### PR TITLE
Added additional error handling for new response formats

### PR DESCRIPTION
**Pull request by xadamxk**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,4 +8,4 @@ requests <3
 requests_ntlm <2
 lxml <5
 pyotp <3
-urllib3<1.25,>=1.20
```
Comment (cshamrick) [inline requirements.txt:11]: @xadamxk why was this removed?

Comment (cshamrick) [reply requirements.txt:11]: More specifically, why was the `<1.25` constraint removed?

Comment (xadamxk) [reply requirements.txt:11]: I was getting conflicts with boto3, since it has different requirements for urllib3, they were conflicting.

Comment (phillipjf) [reply requirements.txt:11]: Python 3.8 compatibility:
```
ERROR: botocore 1.19.6 has requirement urllib3<1.26,>=1.25.4; python_version != "3.4", but you'll have urllib3 1.24.3 which is incompatible.
```

**Review by phillipjf — APPROVED** [review]

**Review by cshamrick — APPROVED** [review]
